src/accessories/dzk-bacnet.ts

```diff
@@ -0,0 +1,991 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+/*
+ *	Copyright (c) 2022 Eric L. Hernes <eric@hernes.ws>
+ *
+ *Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+ *
+ *1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+ *2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+ *THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import {
+  Service,
+  PlatformAccessory,
+  CharacteristicGetCallback,
+  CharacteristicValue,
+  CharacteristicSetCallback,
+  Logger,
+} from "homebridge";
+import bacnet from "bacstack";
+import { BachomeHomebridgePlatform } from "../platform";
+import { objectStringParser } from "../bacnet/parser";
+import {
+  asyncReadPresentValue,
+  asyncWritePresentValue,
+  ValueObject,
+} from "../bacnet/bacnet";
+
+/* XXX-ELH:
+ * it would be really convenient if these were exported from hap-nodejs (hap-nodejs/dist/lib/definitions/CharacteristicDefinitions.d.ts)
+ * But since they are not, we copy here so that we can make the code read a little more symbolic
+ */
+
+/*
+import {
+    TemperatureDisplayUnits,
+    CurrentHeatingCoolingState,
+    TargetHeatingCoolingState,
+} from "hap-nodejs";
+*/
+
+/**
+ * Characteristic "Temperature Display Units"
+ */
+class TemperatureDisplayUnits {
+  static readonly CELSIUS = 0;
+  static readonly FAHRENHEIT = 1;
+}
+/**
+ * Characteristic "Current Heating Cooling State"
+ */
+class CurrentHeatingCoolingState {
+  static readonly OFF = 0;
+  static readonly HEAT = 1;
+  static readonly COOL = 2;
+}
+/**
+ * Characteristic "Target Heating Cooling State"
+ */
+class TargetHeatingCoolingState {
+  static readonly OFF = 0;
+  static readonly HEAT = 1;
+  static readonly COOL = 2;
+  static readonly AUTO = 3;
+}
+
+/**
+ * Characteristic "Current Fan State"
+ */
+class CurrentFanState {
+  static readonly INACTIVE = 0;
+  static readonly IDLE = 1;
+  static readonly BLOWING_AIR = 2;
+}
+/**
+ * Characteristic "Target Fan State"
+ */
+class TargetFanState {
```

Comment:
This seems to not be used. Only in commented out code. Is this meant to be removed oder is the commented out code meant to be not commented out?